src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -98,7 +98,9 @@ def load_arguments(self, _):
     with self.argument_context('resource create') as c:
         c.argument('resource_id', options_list=['--id'], help='Resource ID.', action=None)
         c.argument('properties', options_list=['--properties', '-p'], help='a JSON-formatted string containing resource properties')
-        c.argument('is_full_object', action='store_true', help='Indicates that the properties object includes other options such as location, tags, sku, and/or plan.')
+        c.argument('is_full_object', action='store_true', help='Indicate that the properties object includes other options such as location, tags, sku, and/or plan.')
+        c.argument('latest_version', is_preview=True, action='store_true',
```

Comment:
Maybe `allow_preview` is more reasonable?

```suggestion
        c.argument('allow_preview', is_preview=True, action='store_true',
```

Reply:
In fact, in the original logic, if there is no stable version, the latest preview version will be taken. 
I'm not sure whether using this name will make users feel that if they don't pass this parameter, they can't use preview version.

How about `--latest-include-preview\-lp`?